Lib/fontmake/font_project.py

```diff
@@ -951,7 +951,8 @@ def run_from_designspace(
             preFilters, postFilters = loadFilters(designspace)
             filters = preFilters + postFilters
 
-        source_fonts = [source.font for source in designspace.sources]
+        designspace_copy = self._load_designspace_sources(designspace)
```

Comment:
I wonder why we need to load designspace sources twice..

Reply:
The "interpolate layout" check got upset about something. I don't really understand it: `font["GPOS"] = gpos_src["GPOS"]` died because `gpos_src` didn't have a "GPOS" key.

Reply:
i'd prefer we get to the bottom of this and try to avoid the reload; if we can't we should add a comment explaining why we do that

Reply:
Got it. The interpolate-layout stuff wasn't loading up the binary fonts for the `interpolate_layout_from` designspace because it already had (UFO) fonts in place.

Reply:
Thanks!